JamitLabs/JamitFoundation
# Add collapsible table view cell

# Description
As we do not only need a collapsible cell itself but also want to be able to use it in a table view this MR adds a container table view cell which holds a collapsible view to simply implement the behaviour.

# How to test it
Open the sample app and select "CollapsibleView" as well as "CollapsibleTableViewController" and expand and collapse the items accordingly.

Check the Xcode log output to verify that there do not occur any layout constraint issues.
